apps/researcher/cypress/e2e/community.cy.ts

```diff
@@ -42,3 +42,60 @@ describe('Community details page', () => {
     cy.getBySel('community-name').should('not.exist');
   });
 });
+
+describe('Communities page logged in', () => {
+  beforeEach(() => {
+    cy.session('signed-in', () => cy.signIn());
+  });
+
+  it("opens the 'add community' modal", () => {
+    cy.visit('/en/communities', {
+      failOnStatusCode: false,
+    });
+
+    cy.getBySel('add-community').click();
+    // I can't add test ids to the modal, so I'm using the h1
```

Comment:
Minor: who is 'I' and "I'm"? Perhaps use a general 'we' or an abstract "It's not possible to add test IDs to the model, so we're using the h1 instead"?